proposals/0030-dxil-vectors.md

```diff
@@ -132,6 +132,31 @@ The scalar variants of these DXIL intrinsics will remain unchanged and can be us
 This means that the same language-level vector (of any length) could be used
  in scalarized operations and native vector operations even within the same shader.
 
+### New DXIL Intrinsics
+
+Modern LLVM IR natively supports vector reduction intrinsics but DXIL currently lacks similiar
+capabilities. The following reduction intrinsics are therefore backported as new DXIL Intrinsics.
+
+#### Boolean Vector Reduction Intrinsics
+
+**Vector Reduce And**
+
+Bitwise AND reduction  of the vector returning a scalar. Return type matches vector element type.
```

Comment:
```suggestion
Bitwise AND reduction of the vector returning a scalar. Return type matches vector element type.
```

Reply:
ugh my keyboard is so annoying sometimes